int64 goal_position
int64 goal_velocity
